include/tvm/runtime/logging.h

```diff
@@ -395,10 +399,13 @@ class LogMessageVoidify {
 inline bool DebugLoggingEnabled() {
   static int state = 0;
   if (state == 0) {
-    if (auto var = std::getenv("TVM_LOG_DEBUG")) {
-      if (std::string(var) == "1") {
+    if (const char* var = std::getenv("TVM_LOG_DEBUG")) {
```

Comment:
seems like change not necessary, should keep no change "auto var = std::getenv("TVM_LOG_DEBUG")"

Reply:
We don't have a consistent style on auto, but generally I spell out the types except for auto x = make_object<X>(...), auto f = [...](...) {...}, and iterators.